pkg/registry/apis/provisioning/jobs/driver.go

```diff
@@ -122,7 +122,7 @@ func (d *jobDriver) Run(ctx context.Context) {
 }
 
 func (d *jobDriver) startDriving(ctx context.Context) {
-	timeoutCtx, cancel := context.WithTimeout(ctx, d.timeout)
+	timeoutCtx, cancel := context.WithTimeout(ctx, 1*time.Hour) // ???? currently 28s? d.timeout)
```

Comment:
https://github.com/grafana/grafana/blob/main/pkg/registry/apis/provisioning/register.go#L586

seems like we need something significantly longer -- maybe we should default to 20mins?

(ops takes a few hours)

Reply:
let's put 20 minutes. We should also catch it and complete the job as TIMEOUT error or state and move it to historical jobs.

Reply:
removed from this PR -- lets fix the driver looping/timeout issues in its own PR